examples/js/loaders/MTLLoader.js

```diff
@@ -464,7 +464,16 @@ THREE.MTLLoader.MaterialCreator.prototype = {
 
 					if ( n > 0 ) {
 
-						params.opacity = 1 - n;
+						if ( this.options && this.options.TrOneIsOpaque ) {
+
+							params.opacity = n;
```

Comment:
What is `params.transparent` set to when Tr is 1?

Reply:
Emmmm... good comment. I didn't notice it. fix soon

Reply:
Emmmm... good comment. I didn't notice it. fix soon

Reply:
updated.